libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper-advanced/custom-stepper.component.html

```diff
@@ -0,0 +1,116 @@
+<div class="grid-row">
+  <div class="desktop:grid-col-4 tablet-lg:grid-col-12 mobile-lg:grid-col-12 margin-bottom-3 margin-right-2">
+    <sds-side-toolbar [responsiveButtonText]="'Select Step'" (responsiveDialog)="onDialogOpen($event)">
+      <ng-template #toolbarContent>
+        <div class="sds-card" *ngIf="!!responseDialog">
+          <div class="sds-card__header sds-card__header--accent-cool grid-row">
+            <button
+              id="cancelButton"
+              class="sds-button sds-button--circular sds-button--white"
+              (click)="onCancelClicked()"
+              aria-label="Cancel Select Step"
+            >
+              <usa-icon [icon]="'chevron-left'"></usa-icon>
+            </button>
+            <div class="margin-x-auto">
+              <h2 class="sds-card__title margin-top-1 margin-left-neg-2">Select Step</h2>
+            </div>
+          </div>
+        </div>
+
+        <div class="sds-card">
+          <div class="sds-card__body sds-card__body--accent-cool">
+            <ul class="usa-sidenav usa-sidenav--styled">
+              <li
+                *ngFor="let step of stepTemplates; let i = index"
+                class="usa-sidenav__item"
+                [ngClass]="{
+                  'usa-current': selectedStep?.id === step.id
+                }"
+              >
+                <ng-container [ngTemplateOutlet]="sidenavItem" [ngTemplateOutletContext]="{ $implicit: step }">
+                </ng-container>
+                <ng-container [ngTemplateOutlet]="subPanelTemplate" [ngTemplateOutletContext]="{ $implicit: step }">
+                </ng-container>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </ng-template>
+    </sds-side-toolbar>
+  </div>
+
+  <div class="grid-col-fill">
+    <div [ngTemplateOutlet]="selectedStep ? selectedStep.content : null"></div>
+  </div>
+</div>
+<hr style="width: 100%;" class="fine" />
```

Comment:
Can you move this style to scss? Christy has asked in the past to not include styles since teams may copy the HTML to get started and end up with extra styling in their page that is not obvious from looking at their stylesheets.

Reply:
removed from here and added to styles array in the .ts file